### DISCUSSION TITLE
Airbyte opens too many connections in my external database

### DISCUSSION THREAD

**Discussion by martinbombin** [airbytehq/airbyte#48342] [Platform General Questions]
Hello, I have deployed Airbyte in my Kubernetes cluster by using Helm chart and I realized that it is opening way more connections to my database than I expected. Without making any call to Airbyte it already opens +40 connections to my Postgres database, after a while, it reaches +60 (still no calls to Airbyte were done).

Is there a way to limit the number of connections or reduce those numbers?

**Comment by karolur** [comment]
hi @martinbombin, did you every found a solution? I'm dealing with the same issue.

[a comment by an automated account is not shown]
